src/main/java/org/truffleruby/core/exception/ExceptionNodes.java

```diff
@@ -142,6 +142,16 @@ public Object setMessage(DynamicObject error, Object message) {
 
     }
 
+    @Primitive(name = "exception_formatter")
+    public abstract static class FormatterPrimitiveNode extends CoreMethodArrayArgumentsNode {
```

Comment:
This should be a PrimitiveArrayArgumentsNode (no semantics difference, just consistency)